scripts/prod_stag_sanity_check.py

```diff
@@ -0,0 +1,72 @@
+import json
+from math import isclose
+import os
+import random
+
+import boto3
+from helpers.enums import Network
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
+def download_boosts(chain: str, bucket: str):
+    s3 = boto3.client(
+        "s3",
+        aws_access_key_id=os.environ["AWS_ACCESS_KEY_ID"],
+        aws_secret_access_key=os.environ["AWS_SECRET_ACCESS_KEY"],
+    )
+
+    print("DOWNLOADING BOOST")
+    chain_ids = {
```

Comment:
should we store this dict in constants? that way we can get rid of the web3.chainid call eventually